rounds.lua

```diff
@@ -562,11 +562,11 @@ function Rounds:BeginRound(bot_scripts)
         }
     )
 
-    for candidate_num, _ in pairs(Candidates) do
+    for candidate_num, candidate_name in pairs(Candidates) do
         local hero = self.heros[candidate_num]
         if hero then
             local script = bot_scripts[candidate_num]
-            BotScriptEnv:AttachScriptOnUnit(hero, script)
+            BotScriptEnv:AttachScriptOnUnit(hero, script, candidate_name)
```

Comment:
The function `AttachScriptOnUnit` prototype does not have an argument `candidcate_name`.